types/net_matches.d.ts

```diff
@@ -0,0 +1,16 @@
+import { NetAllFields } from './net_fields';
+import { TriggerTypes } from './net_trigger';
+
+export type Matches = { [s in T]?: string };
+export type ToMatches<F> = Record<keyof F, string>;
```

Comment:
```suggestion
export type ToMatches<Fields> = Record<keyof Fields, string>;
```

Too many single letters in this file for my brain.  :)